cistem/protocols.conf

```diff
@@ -1,34 +1,35 @@
 [PROTOCOLS]
-SPA = [
+Protocols SPA = [
```

Comment:
OK, I understand this needs to be called "Protocols SPA" because this category already exists in other plugins so we need to match that, right?

Reply:
Yes, otherwise you will get two "views" called SPA and Protocols SPA, the last one with a single protocol from cistem

Reply:
Essentially protocols.conf from all plugins are merged/combined on startup